src/style.css

```diff
@@ -0,0 +1,128 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-bold";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: 900;
+}
+
+/* html,
+body {
+  margin: 0;
+  padding: 0;
+} */
+
+input::-webkit-search-cancel-button {
+  -webkit-appearance: none;
+}
+
+.big-search {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+}
+
+.small-search {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  height: 42px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+}
+
+/* .search-icon {
+  margin-top: 26px;
+} */
+
+.small-search > .search-label > .search-icon {
+  /* margin: auto; */
+  width: 11px;
+  height: 11px;
+}
+
+.search-label {
+  display: flex;
+  align-items: center;
+  margin-left: 25px;
+}
+
+.small-search > .search-label {
+  display: flex;
+  align-items: center;
+  margin-left: 13px;
+}
+
+.search-text {
+  border: none;
+  outline: none;
+  font-family: "Avenir-bold", sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 16px;
+  margin-left: 17px;
+  padding: 0;
+}
+
+.small-search > .search-text {
+  margin-left: 9px;
+  padding: 0;
+}
+
+.search-text:active {
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.small-search > .search-label > .search-text {
+  border: none;
+  outline: none;
+  font-family: "Avenir-Heavy", sans-serif;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 14px;
+  line-height: 19px;
+  margin-left: 9px;
+}
+
+.search-text::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.small-search > .search-label > .search-text::placeholder {
+  /* font-family: inherit; */
+  font-size: 14px;
+  font-weight: 400px;
+  color: #3d4e61;
+}
+
+.big-search:hover,
+.small-search:hover {
+  border: 1px solid #e1e7ed;
```

Comment:
when you use color more than 1 time create variable and use it instead of e1e7ed